### PR TITLE
Pagination f end not confirmed sightings profile pg

### PR DESCRIPTION
**Pull request by IsmatMahmood1**
https://trello.com/c/zFCsgA13/69-add-pagination-to-the-frontend-of-not-confirmed-sightings-on-the-profile-page

### CODE REVIEW

File: WhaleSpotting/react-app/src/tests/components/Profile.test.tsx [hunk 1]
```diff
@@ -51,6 +67,43 @@ test("When approval selected get data from API and change heading to Your Approv
...
+        const nextPage = screen.getByTestId("next-page");
+        userEvent.click(nextPage);
+        const card = screen.getByTestId("card-component");
+        expect(card).toBeInTheDocument();
+    }, 200);
+
+    setTimeout(() => {
+        expect(fetchPendingSightings).toBeCalled();
+    }, 300);
+
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/tests/components/Profile.test.tsx]: Unneeded new line

File: WhaleSpotting/react-app/src/tests/components/Profile.test.tsx [hunk 2]
```diff
@@ -51,6 +67,43 @@ test("When approval selected get data from API and change heading to Your Approv
     expect(title).toBeInTheDocument();
 });
 
+test("When click next page approvals load new records", () => {
+    jest.mock("../../api/apiClient", () => ({
+        __esModule: true,
+        fetchPendingSightings: jest.fn(async (pageNumber: number): Promise<SightingApiModel[]> => {
+            return Promise.resolve([mockexample1]);
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/tests/components/Profile.test.tsx]:
```suggestion
            switch (pageNumber) {
                case 1:
                    return Promise.resolve([mockexample1]);
                case 2:
                    return Promise.resolve([mockexample2]);
                default:
                    return Promise.resolve([]);
            }
```
If you want to check the data changes between pages this is how you would

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/react-app/src/tests/components/Profile.test.tsx (hunk 2 above)
Comment (IsmatMahmood1) [reply WhaleSpotting/react-app/src/tests/components/Profile.test.tsx]: test amended

Comment (LenaESoftwire) [reply WhaleSpotting/react-app/src/tests/components/Profile.test.tsx]: changed the tests

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**